qns/Dockerfile

```
@@ -14,19 +14,15 @@ ENV RUSTUP_HOME=/usr/local/rustup \
     PATH=/usr/local/cargo/bin:$PATH
 
 RUN set -eux; \
```

I'm less of a fan of dropping the checksum, but I can see the value.

```suggestion
RUN \
```

You don't need the set call (unless you want to call `set -o pipefail`).